WORKSPACE

```diff
@@ -109,11 +162,28 @@ load(
     "new_container_load",
 )
 
+# To test the new_container_load rule.
 new_container_load(
     name = "new_pause_tar",
     file = "//testdata:pause.tar",
 )
 
+# Images for testing new_container_load
+new_container_pull(
```

Comment:
Is it used?

Reply:
Saw that in the testdata/BUILD. But this is image is big, we probably don't want to have it here.